spock-core/src/main/java/org/spockframework/mock/runtime/CgLibMockMaker.java

```diff
@@ -0,0 +1,42 @@
+package org.spockframework.mock.runtime;
+
+import org.spockframework.mock.CannotCreateMockException;
+import org.spockframework.util.ReflectionUtil;
+import org.spockframework.util.ThreadSafe;
+import spock.mock.MockMakerId;
+
+import java.lang.reflect.Modifier;
+
+@ThreadSafe
+public class CgLibMockMaker implements IMockMaker {
```

Comment:
Shouldn't it be `Cglib`? The naming is wrong in several places, e.g., `MockMakerId.cglib`

Reply:
Fixed